client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -109,6 +109,16 @@ public final class ApiEndpointsV1
     public static final String ABORT_RESTORE_JOB_ROUTE = RESTORE_JOB_ROUTE + ABORT;
     public static final String RESTORE_JOB_PROGRESS_ROUTE = RESTORE_JOB_ROUTE + PROGRESS;
 
+    // CDC APIs
+    public static final String CDC_PATH = "/cdc";
+    public static final String LIST_ALL_TABLE_IDS_ROUTE = API_V1 + "/table-ids";
+    public static final String LIST_TABLE_IDS_BY_KEYSPACE_ROUTE = LIST_ALL_TABLE_IDS_ROUTE + "/keyspace/" + KEYSPACE_PATH_PARAM;
```

Comment:
let's use keyspaces plural here:
```suggestion
    public static final String LIST_TABLE_IDS_BY_KEYSPACE_ROUTE = LIST_ALL_TABLE_IDS_ROUTE + PER_KEYSPACE;
```